docs/source/usage/picmi/intro.rst

```diff
@@ -150,17 +173,34 @@ Parameters/Methods prefixed with ``picongpu_`` are PIConGPU-exclusive.
 
 - **Species**
 
-  - ``picongpu_ionization_electrons``:
-    Electron species to use for ionization.
-    Optional, will be guessed if possible.
-  - ``picongpu_fully_ionized``:
-    When defining an element (using ``particle_type``) it may or may not be ionizable
+  - ``picongpu_fixed_charge``:
+    When defining an ion species using ``particle_type`` it may or may not be ionizable
+
+    - to **enable** ionization add an ionization model to the Interaction object of the simulation and set the initial charge state using ``charge_state``.
+    - to **disable** ionization set ``picongpu_fixed_charge=True``, this will fix the charge of particles of this species for entire simulation.
 
-    - to **enable** ionization simulation set ``charge_state`` to an integer
-    - to **disable** ionization (ions are only core without electrons) set ``picongpu_fully_ionized=True``
+    ``picongpu_fixed_charge`` maybe combined with ``charge_state`` to control which charge state is to used for the ion species
 
     If neither is set a warning is printed prompting for either of the options above.
 
+- **Interaction**
+  Configuration of the PIC-algorithm extensions, use as follows:
+
+  .. code:: python
+
+    from picongpu import picmi
+    from picongpu.interaction.ionization.fieldionization import ADK, ADKVariant
+
+    e = picmi.Species(name="e", particle_type="electron")
+    nitrogen = picmi.Species(name="nitrogen", particle_type="N", charge_state=2)
+
+    ADK_ionization = ADK(ADK_variant = ADKVariant.LinearPolarization, ion_species = nitrogen, ionization_electron_species=e)
+    interaction = Interaction(ground_state_ionizaion_model_list=[ADK_Ionization])
+
+    sim = picmi.simulation(picongpu_interaction=interaction)
+    sim.add_species(e, ...)
+    sim.add_species(nitrogen, ...)
```

Comment:
Don't place code into the documentation. Place it in an executable example script and pull it in from there via the corresponding `.rst` feature (see how it is solved in https://picongpu--5007.org.readthedocs.build/en/5007/usage/metadata.html).

Reply:
Offline discussion: Resolve after opening an issue.

Reply:
issue #5013